aws-production-2/main.tf

```diff
@@ -103,9 +136,16 @@ module "aws_asg_org" {
   worker_asg_scale_in_threshold = 16
   worker_asg_scale_out_qty = 2
   worker_asg_scale_out_threshold = 8
+  worker_cache_bucket = "${var.worker_org_cache_bucket}"
```

Comment:
are the cache bucket's the beginnings of moving off of quay.io for ASGs?  Or just holding state within AWS?